tests/vmi_cloudinit_test.go

```diff
@@ -191,7 +191,10 @@ var _ = Describe("[rfe_id:151][crit:high][vendor:cnv-qe@redhat.com][level:compon
 						fedoraPassword,
 						sshAuthorizedKey,
 					)
-					vmi := tests.NewRandomVMIWithEphemeralDiskAndConfigDriveUserdataHighMemory(cd.ContainerDiskFor(cd.ContainerDiskFedoraTestTooling), userData)
+					vmi := libvmi.NewFedora(
+						libvmi.WithContainerDisk("disk0", cd.ContainerDiskFor(cd.ContainerDiskFedoraTestTooling)),
+						libvmi.WithCloudInitConfigDriveUserData(userData),
+						libvmi.WithResourceMemory("512M"))
```

Comment:
AFAIU `NewFedora` already uses `ContainerDiskFedoraTestTooling` container disk.
```suggestion
					vmi := libvmi.NewFedora(
						libvmi.WithCloudInitConfigDriveUserData(userData),
						libvmi.WithResourceMemory("512M"))
```

Reply:
yes, and it is also with 512Mi of memory, so i'll remove that as well.
also, `NewFedora()` is configuring:
```
vmi.Spec.Domain.Devices.Rng = &v1.Rng{}
```
to the VMI. does having a random number generator is ok for this test case?

Reply:
I think it does not create any issues here! Thank you